### PR TITLE
Required tasks for completion of the assignment.

### PR DESCRIPTION
**Pull request by Samundra2005**

### CODE REVIEW

File: hw03_samundra.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+#name:Samundra Adhikari
+#username: adhikaris2
+#googledocs: https://docs.google.com/document/d/1Mj5nshMpoanpbSKld58QdK__H9CcjM_WI2m9WeCyrzo/edit#heading=h.66gmzdtk59q2
+# Drawing a cat.
+#Acknowlege: Used ChatGPT to ask questions, and used textbook.
```
Comment (sheggen) [inline hw03_samundra.py:5]: Use the header block formatting provided in the template.

File: hw03_samundra.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+def draw_head(cat, x,y):
+    """ This function draws the head of a cat"""
+    cat.penup()
+    cat.goto(0, -100)
+    cat.pendown()
+    cat.begin_fill()
+    cat.color("#110101")
+    cat.circle(100)
+    cat.end_fill()
+
```
Comment (sheggen) [inline hw03_samundra.py:20]: Check for gray squigglies, which indicate style guidelines. This one says to put two blank lines before each function definition.

File: hw03_samundra.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+#username: adhikaris2
+#googledocs: https://docs.google.com/document/d/1Mj5nshMpoanpbSKld58QdK__H9CcjM_WI2m9WeCyrzo/edit#heading=h.66gmzdtk59q2
+# Drawing a cat.
+#Acknowlege: Used ChatGPT to ask questions, and used textbook.
+
+import turtle
+
+# Draw the cat's head
+
+def draw_head(cat, x,y):
```
Comment (sheggen) [inline hw03_samundra.py:11]: Why do you have input parameters `x` and `y`? They aren't used in this function or most of the ones below, either.

File: hw03_samundra.py (hunk 1 above)
Comment (Samundra2005) [reply hw03_samundra.py]: I have done the formatting as per the template. Thank you!

File: hw03_samundra.py (hunk 2 above)
Comment (Samundra2005) [reply hw03_samundra.py:30]: I have fixed the spacing issues. Thank you!

File: hw03_samundra.py (hunk 3 above)
Comment (Samundra2005) [reply hw03_samundra.py]: I have fixed these issues.

File: hw03_samundra.py (hunk 2 above)
Comment (sheggen) [reply hw03_samundra.py:30]: Check for this same problem other places in the code, such as lines 16-20

File: hw03_samundra.py [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+#################################################################################
+# Acknowledgements: Used ChatGPT to ask questions, and used textbook.
+#
+#
+#################################################################################
+
+import turtle
+
+
+# Draw the cat's head
```
Comment (sheggen) [inline hw03_samundra.py:18]: This should probably in be the docstring for `main()`, which is also missing

Comment (Samundra2005) [reply hw03_samundra.py]: I have corrected the code! Thank you

File: hw03_samundra.py (hunk 2 above)
Comment (sheggen) [reply hw03_samundra.py:27]: Screenshot 2024-02-06 at 2 23 32 PM

**Pull request merged**